### PR TITLE
feat(blog): add 6.5.4 release notes to blog

### PR DESCRIPTION
**Pull request by pswu11**
major: 
- add 6.5.4 release notes to the blog with external link

minor: 
- format (prettier) of previous release post
- add a link to release notes for 6.5 release blog post in case users land on the post directly

### CODE REVIEW

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description: QuestDB 6.5.4 release notes
+keywords: [questdb, release, database, hacktoberfest, opensource]
+tags: [release, product, hacktoberfest, event, community]
+image: /img/blog/2022-10-25/banner.png
+permalink: https://github.com/questdb/questdb/releases/tag/6.5.4
+---
+
+import Screenshot from "@theme/Screenshot"
+
+This is a
```
Comment (amyshwang) [inline blog/2022-10-25-questdb-release-6.5.4.mdx:16]:
```suggestion
Thanks to our
amazing Hacktoberfest contributors, this is a
```

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[special release](https://github.com/questdb/questdb/releases/tag/6.5.4) for our
+amazing Hacktoberfest contributors, including new SQL features with a focus on
+string and math functions, as well as performance and stability improvements
+brought by our core engineers.
+
+<!--truncate-->
+
```
Comment (amyshwang) [inline blog/2022-10-25-questdb-release-6.5.4.mdx:23]: The "read more" links to the GH release note, not the content below?

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+keywords: [questdb, release, database, hacktoberfest, opensource]
+tags: [release, product, hacktoberfest, event, community]
+image: /img/blog/2022-10-25/banner.png
+permalink: https://github.com/questdb/questdb/releases/tag/6.5.4
+---
+
+import Screenshot from "@theme/Screenshot"
+
+This is a
+[special release](https://github.com/questdb/questdb/releases/tag/6.5.4) for our
```
Comment (amyshwang) [inline blog/2022-10-25-questdb-release-6.5.4.mdx:17]: This link is the same as the "read more" in the preview:
image

It is probably too repetitive? Would you consider just use the full content as shown in https://deploy-preview-1154--unruffled-blackwell-31bfb2.netlify.app/blog/2022/10/25/questdb-release-6.5.4?

I am a bit confused by the hyperlink "read more" linking to GH...

Comment (pswu11) [reply blog/2022-10-25-questdb-release-6.5.4.mdx:17]: it's truncated, it's exactly the same content just not full length.

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+amazing Hacktoberfest contributors, including new SQL features with a focus on
```
[the remark this answers is not in the document]
Comment (pswu11) [reply blog/2022-10-25-questdb-release-6.5.4.mdx]: I'll do this 
> Thanks to our amazing Hacktoberfest contributors, this release includes new SQL
> features with a focus on string and math functions, as well as performance and
> stability improvements brought by our core engineers.

File: blog/2022-10-25-questdb-release-6.5.4.mdx (hunk 3 above)
Comment (pswu11) [reply blog/2022-10-25-questdb-release-6.5.4.mdx]: > This link is the same as the "read more" in the preview:

this is how our blog works, look at the current article. :)

Comment (pswu11) [reply blog/2022-10-25-questdb-release-6.5.4.mdx]: Do you mean, we should just not truncate at all?

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- Support using `row_number()` with `ORDER BY`
+- Allow `ALTER DROP` on the latest active partition
+- Improve performance for ILP and `GROUP BY` aggregations
+
+Special thanks to [@ArjunSharda](https://github.com/ArjunSharda),
+[@TheZal](https://github.com/TheZal), [@Ln11211](https://github.com/Ln11211),
+[@ShubhamPalriwala](https://github.com/ShubhamPalriwala),
+[@blackstar-baba](https://github.com/blackstar-baba),
+[@akki1306](https://github.com/akki1306),
+[@DarthBenro008](https://github.com/DarthBenro008),[@bsnxsourav](https://github.com/bsnxsourav),
```
Comment (amyshwang) [inline blog/2022-10-25-questdb-release-6.5.4.mdx:39]:
```suggestion
[@DarthBenro008](https://github.com/DarthBenro008), [@bsnxsourav](https://github.com/bsnxsourav),
```

File: blog/2022-10-25-questdb-release-6.5.4.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+[@sklarsa](https://github.com/sklarsa),
```
Comment (amyshwang) [inline blog/2022-10-25-questdb-release-6.5.4.mdx:40]: This is our own Steve?

**Review by amyshwang — APPROVED** [review]

**Review by tris0laris — APPROVED** [review]

**Pull request merged**